Dockerfile

```diff
@@ -1,11 +1,13 @@
-FROM golang:1.14 as builder
+FROM --platform=$BUILDPLATFORM golang:1.14 as builder
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
```

Comment:
I don't think you need to set ARG `TARGETPLATFORM` or `BUILDPLATFORM` since it's part of the automatic platform arg and `TARGETPLATFORM` is inferred once you set `--platform`. Though in this case would you not want --platform=$TARGETPLATFORM since this will be the platform from the arg `--platform linux/$(ARCH) ` instead of the build node's platform from `BUILDPLATFORM`?
https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope

Reply:
You're right. I'll submit a fix.